smartling-files-api/src/main/java/com/smartling/api/files/v2/FilesApi.java

```diff
@@ -41,79 +43,82 @@
 
 @Produces(APPLICATION_JSON)
 @Consumes(APPLICATION_JSON)
-@Path("/files-api/v2")
 @DetailedErrorMessage(args = "fileUri")
 public interface FilesApi extends AutoCloseable
 {
     @POST
-    @Path("/projects/{projectId}/file")
+    @Path("/files-api/v2/projects/{projectId}/file")
     @Consumes(MULTIPART_FORM_DATA)
     UploadFileResponse uploadFile(@PathParam("projectId") String projectId, @MultipartForm UploadFilePTO uploadFilePTO);
 
     @GET
-    @Path("/projects/{projectId}/file")
+    @Path("/files-api/v2/projects/{projectId}/file")
     @Produces(WILDCARD)
     InputStream downloadSourceFile(@PathParam("projectId") String projectId, @QueryParam("fileUri") String fileUri);
 
     @GET
-    @Path("/projects/{projectId}/file/status")
+    @Path("/files-api/v2/projects/{projectId}/file/status")
     FileStatusResponse getFileStatus(@PathParam("projectId") String projectId, @QueryParam("fileUri") String fileUri);
 
     @GET
-    @Path("/projects/{projectId}/locales/{localeId}/file/status")
+    @Path("/files-api/v2/projects/{projectId}/locales/{localeId}/file/status")
     FileLocaleStatusResponse getFileLocaleStatus(@PathParam("projectId") String projectId, @PathParam("localeId") String localeId, @QueryParam("fileUri") String fileUri);
 
     @GET
-    @Path("/projects/{projectId}/locales/{localeId}/file")
+    @Path("/files-api/v2/projects/{projectId}/locales/{localeId}/file")
     @Produces(WILDCARD)
     InputStream downloadTranslatedFile(@PathParam("projectId") String projectId, @PathParam("localeId") String localeId, @BeanParam DownloadTranslationPTO downloadTranslationPTO);
 
     @GET
-    @Path("/projects/{projectId}/locales/{localeId}/file")
+    @Path("/files-api/v2/projects/{projectId}/locales/{localeId}/file")
     @Produces(MULTIPART_MIXED)
     TranslatedFileMultipart downloadTranslatedFileMultipart(@PathParam("projectId") String projectId, @PathParam("localeId") String localeId, @BeanParam DownloadTranslationPTO downloadTranslationPTO);
 
     @GET
-    @Path("/projects/{projectId}/locales/all/file/zip")
+    @Path("/files-api/v2/projects/{projectId}/locales/all/file/zip")
     @Produces(WILDCARD)
     InputStream downloadAllFileTranslations(@PathParam("projectId") String projectId, @BeanParam DownloadAllFileTranslationsPTO downloadAllFileTranslationsPTO);
 
     @GET
-    @Path("/projects/{projectId}/files/zip")
+    @Path("/files-api/v2/projects/{projectId}/files/zip")
     @Produces(WILDCARD)
     InputStream downloadMultipleFileTranslations(@PathParam("projectId") String projectId, @BeanParam DownloadMultipleTranslationsPTO downloadMultipleTranslationsPTO);
 
     @GET
-    @Path("/projects/{projectId}/files/list")
+    @Path("/files-api/v2/projects/{projectId}/files/list")
     ListResponse<FileItemPTO> getFilesList(@PathParam("projectId") String projectId, @BeanParam GetFilesListPTO getFilesListPTO);
 
+    @GET
+    @Path("/published-files-api/v2/projects/{projectId}/files/list/recently-published")
```

Comment:
As far as prefix of the endpoint is `published-files-api` it seemed we should have them in `PublishedFilesApi.java` file.